flink-metrics/flink-metrics-core/pom.xml

```diff
@@ -35,6 +35,12 @@ under the License.
 	<packaging>jar</packaging>
 
 	<dependencies>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-annotations</artifactId>
```

Comment:
For historic reasons flink-metrics-core has no further dependencies, and ideally we change that as an independent effort